templates/zerver/base.html

```diff
@@ -19,6 +19,9 @@
     <link href="/static/favicon.ico?v=2" rel="shortcut icon">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
+    <link href="/static/node_modules/katex/dist/katex.min.css" rel="stylesheet">
+    <script src="/static/node_modules/katex/dist/katex.min.js"></script>
```

Comment:
So this won't work in production; you'll need to move these to `PIPELINE` and JS_SPECS` in zproject/settings.py so that they get put in the right place (`node_modules` isn't shipped to or served in production, so everything useful needs to be installed somewhere else).  

I think there's an interesting question for whether we to just add KaTeX to `app.css` / `app.js` or try to do something that only loads these if they're needed (to minimize page weight).  I think I'd probably start with just including them unconditionally (aka the easy thing), and we can measure the impact on the resulting static file sizes.